api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -178,4 +179,18 @@ public void validate_shouldPassForAValidAllergy() {
 		
 		assertFalse(errors.hasErrors());
 	}
+
+	@Test
+	public void valid_shouldRejectNumericReactionValue() throws Exception {
```

Comment:
do you want to name your test method like the method you're testing ? e.g
`public void validate_shouldRejectNumericReactionValue() throws Exception {}`

Reply:
oh...sorry....let me remove throws exception.........that's not right ......thanks @tendomart